com.note_slayer.NoteSlayer.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: com.note_slayer.NoteSlayer
+base-version: "23.08"
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: noteslayer
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=home
```

Comment:
Why do you need full home? Does it support filechooser portals?

Reply:
Yes

Reply:
Then why?

Reply:
i have a feature where user export excel file so he need to choose file directory

Reply:
If you support file choosers this should work automatically?